common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2020,11 +2020,23 @@
     ],
     "sqlState" : "XX000"
   },
-  "INTERVAL_ARITHMETIC_OVERFLOW" : {
-    "message" : [
-      "<message>.<alternative>"
+  "INTERVAL_ARITHMETIC_OVERFLOW": {
+    "message": [
+      "Integer overflow while creating interval."
     ],
-    "sqlState" : "22015"
+    "subClass": {
+      "NUMERIC": {
+        "message": [
+          "<message>.<alternative>."
+        ]
+      },
+      "DATETIME": {
+        "message": [
+          "Too big interval for datetime, check how you create it."
```

Comment:
Ideally I want to write here something like: "Interval is too big, it only fits 2^31-1 days (if it's make_ym_interval) and 2^63-1 microseconds (if it's make_dt_interval)".
But I don't see something similar to this in that file.

Reply:
You can say something like: Number of days <actualNumDays> exceeds the allowed limit of integer value. Is it only while creating intervals that this can happen, I got the idea that it can happen with arithmetic operations as well. So I would say integer overflow while operating with intervals.

Reply:
This change for subClass requires addition of new parameter to the error, which is fine as well.